distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -590,7 +594,7 @@ generate_create_table dialect name columns primary_key temporary =
     column_definitions = columns.map (generate_column_description dialect)
     modifiers = if primary_key.is_nothing then [] else
         [SQL_Builder.code ", PRIMARY KEY (" ++ SQL_Builder.join ", " (primary_key.map dialect.wrap_identifier) ++ ")"]
-    table_type = if temporary then "TEMPORARY TABLE" else "TABLE"
+    table_type = if temporary && dialect.suppports_temporary_table_syntax then "TEMPORARY TABLE" else "TABLE"
```

Comment:
It feels to me that we should fail saying that this is not supported. It may trip too many tests here, but maybe at least a warning??

I'm feeling rather anxious that we are telling the user that we are creating a temporary table but it is actually a permanent one - this can very easily lead to users polluting their DBs with tons of garbage tables by accident.

Reply:
Will fix as a quick follow up https://github.com/enso-org/enso/issues/10711